src/main/java/CarRacing/Domain/Position.java

```diff
@@ -0,0 +1,34 @@
+package CarRacing.Domain;
+
+public class Position {
+    private static final int Position_min = 0;
```

Comment:
static 변수의 경우 대문자를 쓰는 컨벤션이 있습니다 :3
```suggestion
    private static final int POSITION_MIN = 0;
```